app/models/ingredient_mention.rb

```diff
@@ -0,0 +1,3 @@
+class IngredientMention < ApplicationRecord
+	validates :name, uniqueness: true
```

Comment:
this indentation is not consistent with the rest of the app; too deep

Reply:
I fixed it and pushed it, but I dont know where to see that change take place because it says everything is up to date in terminal